src/contexts/PredictoorsContext.tsx

```diff
@@ -356,7 +353,8 @@ export const PredictoorsProvider: React.FC<TPredictoorsContextProps> = ({
         const newCurrentEpoch = Math.floor(currentTs / SPE)
         if (
           currentTs - lastCheckedEpoch.current * SPE <
-          SPE + PREDICTION_FETCH_EPOCHS_DELAY
+            SPE + PREDICTION_FETCH_EPOCHS_DELAY &&
+          timeFrameInterval == lastTimeframe
```

Comment:
getting the value of a ref need to be with the "current" property. `lastTimeframe.current`